synapse/config/logger.py

```diff
@@ -348,3 +350,4 @@ def setup_logging(
     )
     logging.info("Server hostname: %s", config.server.server_name)
     logging.info("Instance name: %s", hs.get_instance_name())
+    logging.info("Twisted reactor: %s", type(reactor).__name__)
```

Comment:
I'm assuming that we need to install the asyncio reactor first before we do this? But that should have already happened at import time.

Reply:
That's right, the reactor gets installed quite early